setup/web.sh

```diff
@@ -81,18 +80,17 @@ chmod a+r /var/lib/mailinabox/mobileconfig.xml
 # The format of the file is documented at:
 # https://wiki.mozilla.org/Thunderbird:Autoconfiguration:ConfigFileFormat
 # and https://developer.mozilla.org/en-US/docs/Mozilla/Thunderbird/Autoconfiguration/FileFormat/HowTo.
-cat conf/mozilla-autoconfig.xml \
-	| sed "s/PRIMARY_HOSTNAME/$PRIMARY_HOSTNAME/" \
+< conf/mozilla-autoconfig.xml sed "s/PRIMARY_HOSTNAME/$PRIMARY_HOSTNAME/" \
```

Comment:
In this case my preference is `sed in \n > out` or keeping the cat for consistency. Would defer to @JoshData.

Reply:
If you want to fix the bug, but keep it as two commands:
```
cp conf/mozilla-autoconfig.xml /var/lib/mailinabox/mozilla-autoconfig.xml
sed -i "s/PRIMARY_HOSTNAME/$PRIMARY_HOSTNAME/" /var/lib/mailinabox/mozilla-autoconfig.xml
```
Or, possibly your suggestion above: https://github.com/mail-in-a-box/mailinabox/pull/1457#discussion_r230141268.